server/app/services/email/EmailSendClient.java

```diff
@@ -0,0 +1,25 @@
+package services.email;
+
+import com.google.common.collect.ImmutableList;
+
+/** Interface for sending emails from CiviForm. */
+public interface EmailSendClient {
+
+  /**
+   * Sends an email from CiviForm with one recipient.
+   *
+   * @param toAddress A single address of where the email should be sent.
```

Comment:
Blocking: How does the caller know whether the email was sent successfully, or resulted in an error?

Reply:
We have some monitoring for this and the fail count will increase if there was an error. https://github.com/civiform/civiform/blob/e0f3efed9d4951a49e5df22818fb951c59204e1f/server/app/services/email/aws/SimpleEmail.java#L117-L121